async/abortable.ts

```diff
@@ -78,7 +101,11 @@ export function abortable<T>(
  * const p = abortablePromise(request, c.signal);
  *
  * // The below throws if the request didn't resolve in 100ms
- * await p;
+ * try {
+ *   await p;
+ * } catch (e) {
+ *   console.error(e);
+ * }
```

Comment:
Not sure if these try/catches are worth doing because they add a bit of noise.

Reply:
@dsherret its either doing those, or adding `no-eval`, as else the doc test task will fail. I don't mind either way

Reply:
discussed with @iuioiua, will go with no-eval